ofprotocol/ofp13/ofp13_parser.go

```diff
@@ -1402,7 +1403,12 @@ func (m *OfpMatch) Parse(packet []byte) {
 	index += 2
 
 	for index < (int(m.Length) - 4) {
-		mf := parseOxmField(packet[index:])
+		mf, err := parseOxmField(packet[index:])
+		if err != nil {
+			fmt.Printf("Error on parsing oxm field: %v", err)
+			index += (int)(packet[index+3]) + 4  // 4 -sizeof(header)
+			continue
```

Comment:
why is it okay to continue here?

Reply:
When the controller crashes here we found in debugging and in a dump of traffic between vswitch and controller unsupported by controller OXM Class: OFPXMC_NXM_1. Documentation(https://www.opennetworking.org/wp-content/uploads/2014/10/openflow-spec-v1.3.0.pdf) says(page 40) "classes OFPXMC_NXM_0 and OFPXMC_NXM_1 are reserved for backward compatibility". So we can ignore those classes.